src/parse.c

```diff
@@ -359,8 +375,7 @@ static int container_parse_fsmap(struct hyper_container *c, char *json, jsmntok_
 					c->maps[j].docker = 1;
 				dprintf(stdout, "maps %d docker volume %d\n", j, c->maps[j].docker);
 			} else {
-				dprintf(stdout, "in maps incorrect %s\n",
-					json_token_str(json, &toks[i]));
+				hyper_print_unknown_section(json, &toks[i]);
 				return -1;
```

Comment:
actually, the best way to handle unknown keys is ignoring them.
let runv handle the api compatibility.
it will make the api-updating easier.